### DISCUSSION TITLE
article/notion-next-themes

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#81] [Announcements]
# article/notion-next-themes

这是一个由NotionNext生成的站点

https://docs.tangly1024.com/article/notion-next-themes

**Comment by gaoran1209** [comment]
hi，请问，我使用 hexo 主题的时候发现文章列表宽度无法对齐，有解决办法吗？
看了半天配置项，不知道是改动了那个参数导致的。
